src/webui/www/public/scripts/client.js

```diff
@@ -310,13 +312,19 @@ window.addEvent('load', function () {
                         update_categories = true;
                     }
                     if (response['torrents']) {
+                        var newTorrentInList = false;
                         for (var key in response['torrents']) {
                             response['torrents'][key]['hash'] = key;
                             response['torrents'][key]['rowId'] = key;
                             torrentsTable.updateRowData(response['torrents'][key]);
                             if (addTorrentToCategoryList(response['torrents'][key]))
                                 update_categories = true;
+                            if (response['torrents'][key]['name'])
+                                newTorrentInList = true;
```

Comment:
Correct, it needs to bind to any new rows that are created. It should also un-bind from rows as they are deleted.